README.rst

```diff
@@ -1,5 +1,6 @@
 .. _Thingy: https://github.com/Refty/thingy
 .. _PyMongo: https://github.com/mongodb/mongo-python-driver
+.. _Mongomock: https://github.com/mongomock/mongomock
```

Comment:
It could be useful to show how to use an alternative backend in the documentation.